spec/takeaway_spec.rb

```diff
@@ -0,0 +1,92 @@
+require 'takeaway'
+
+describe Takeaway do
+  let(:menu) { double :menu }
+  let(:shop) { described_class.new(menu) }
+  let(:name_one) { "name one" }
+  let(:price_one) { "£0.01" }
+  let(:name_two) { "name two" }
+  let(:price_two) { "£0.02" }
```

Comment:
These make the tests harder to read IMO, being up there. Nothing wrong with having hard coded strings in your tests. It may even be better.